src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+    <a href="https://glitch.com/edit/#!/snap?path=carousel.html:1:0"
+       target="_blank">Source</a>
+  </figcaption>
+</figure>
+
+
+### Example - Journeyed product page
+
+Another common case that can benefit from scroll snapping are pages with
+multiple logical sections that are vertically scrolled through, e.g., a typical
+product page. ``scroll-snap-type: y proximity;` is a `more natural fit for cases
+like this. It does not interfere when user scrolls to the middle of a particular
+section but also snaps and brings attention to a new section when they scroll
+close enough to it.
+
+Here is how this can be achieved:
+
+
+```
+<style>
+article {
+  scroll-snap-type: y proximity;
+  /* Reserve space for header plus some extra space for sneak peeking. */
+  scroll-padding-top: 15vh;
+  overflow-y: scroll;
+}
+section {
+  /* Snap align start. */
+  scroll-snap-align: start;
+}
+header {
+  position: fixed;
+  height: 10vh;
+}
+</style>
+
+<article>
+  <header> Header </header>
+  <section> Section One </section>
+  <section> Section Two </section>
+  <section> Section Three </section>
+</article>
+```
+
+
+#### Scroll padding and margin
+
+Our product page has a fixed position top header. Our design also asked for some
+of the top section to remain visible when scroll container is snapped in order
+to provide a design cue to users about the content above.
+
+
+`scroll-padding` is a new css property that can be used to adjust the effective
+viewable region of scroll container.  This region is also known as snapport and
+is used when calculating scroll snap alignments. The property defines an inset
+against the scroll container's padding box. In our example 15vh additional inset
+was added to the top which instructs the browser to consider a lower position,
+15vh below the top edge of the scroll container, as its vertical start edge for
+scroll snapping. When snapping, the start edge of the snap target element will
+become flushed with this new position thus leaving space above.
+
+`scroll-margin` defines the outset amount used to adjust the snap target
+effective box similar to how `scroll-padding` functions on snap scroll
+container.
+
+You may have noticed that these two properties do not have the word "`snap`" in
+them. This is intentional as they actually modify the box for all relevant
+scroll operations and are not just scroll snapping. For example Chrome takes
+them into account when calculating page size for paging scroll operations such
+as PageDown and PageUp and also when calculating scroll amount for
+`Element.scrollIntoView()` operation.
+
+
+<figure>
+  <a href="/web/updates/images/2018/07/css-scroll-snap/product-page.mp4"
+     target="_blank">
+    <video src="/web/updates/images/2018/07/css-scroll-snap/product-page.mp4"
+           autoplay muted playsinline loop controls alt="Product page demo" height="600"></video>
+  </a>
+  <figcaption>
+    <a href="https://snap.glitch.me/product.html"
```

Comment:
Would you please add `<html lang="en">` to the demo, Chrome is prompting me to translate the page ;)

Reply:
Done